src/register/Register.jsx

```diff
@@ -156,6 +156,7 @@ export default function Register() {
                                     placeholder="Password"
                                     minLength={8}
                                     maxLength={64}
+                                    pattern="^[^\s].[a-zA-Z0-9/_]+$"
```

Comment:
What about special characters in the password? It must not give validation when special characters like `@`is present in password.

Reply:
@Rahulm2310 and @codesankalp , here're my point of view on this:
1. the accepted regex pattern for validation needs to be consistent across backend and frontend sides of the application. In our case, since BIT is following MS backend rules on accepted regex for user's Registration, BIT backend and frontend must follow the same rules stated [here](https://github.com/anitab-org/mentorship-backend/blob/develop/app/utils/validation_utils.py).
2. @isabelcosta in [this zulip post](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system/topic/Registration.20problem/near/189269896) had stated the reason behind why atm we don't (and perhaps better not) use regex pattern for password (which is to make it easier for developer while we're still in the development stage - a different story when MS is in the production stage). 
3. MS android as MS frontend also faces the same issue with Registration input validation which I raised [here](https://github.com/anitab-org/mentorship-android/issues/710). I also opened this Zulip [discussion](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system/topic/Common.20Input.20validations/near/198847620) to start discussing potential solution. We still waiting for others' to give their feedback. 

Let me know what you think of this.

Reply:
I think we need to limit the scope of this pr to address only the bug stated in the issue #202 which is if the input consists of only spaces then show warning. We can use a pre-check and only do field validation check using the existing regex pattern in the form (which I believe already matched to the patterns used on MS backend) if the input contains alphabets.